folium/raster_layers.py

```diff
@@ -73,33 +76,29 @@ def __init__(self, tiles='OpenStreetMap', min_zoom=0, max_zoom=18,
         self._name = 'TileLayer'
         self._env = ENV
 
-        options = {
-            'minZoom': min_zoom,
-            'maxZoom': max_zoom,
-            'noWrap': no_wrap,
-            'attribution': attr,
-            'subdomains': subdomains,
-            'detectRetina': detect_retina,
-        }
+        options = {'minZoom': min_zoom, 'maxZoom': max_zoom, 'noWrap': no_wrap,
+                   'attribution': attr, 'subdomains': subdomains,
+                   'detectRetina': detect_retina}
```

Comment:
The 1 option per line is by design, for readability. Do you mind reverting that change?

Reply:
Sure no problem